### ISSUE TITLE
425307D1 - The Elder Scrolls 4: Oblivion - Game of the Year Edition

### ISSUE THREAD

**Issue by TotalCaesar659** [xenia-project/game-compatibility#576]
# Disk 1

[Marketplace](http://marketplace.xbox.com/en-US/Product/Oblivion/66acd000-77fe-1000-9115-d802425307d1)

Tested on (https://github.com/benvanik/xenia/commit/abf162c0e32889b37461374b1d25ef9b5bba2648)

# Issues:
Instant crash.

# Log:
[Log file](https://drive.google.com/file/d/0B7KpdfS_wYnZei1VRFBvUXVlQzg/view)

# Screenshot(s):
![0000](https://cloud.githubusercontent.com/assets/14265316/17651359/38616b2e-626e-11e6-9ac0-46a5709652f4.jpg)

# Labels:
(state-crash)

**Comment by shoopdahoop22** [comment]
How come it's running ingame here?

https://youtu.be/KlLoF81vDH0

Is it an issue only present in the GOTY version?  Or is it a regression?

**Comment by Razzile** [comment]
seems like failed allocations from the log, but i'm no expert with xenia

**Comment by theoldsport** [comment]
That allocation issue may be similar to what I'm still experiencing with CoD 4 (and some other games). For most people on Nvidia, there are no problems running certain games. Yet despite using an Nvidia gpu and running the same build, these titles won't boot at all (even though they should).

**Comment by AussiePlick** [comment]
Yes, I also have crash even before black screen or anything noticeable.
I have a ASUS STRIX GTX 970 and using latest build off Appveyer.
Also have latest Nvidia drivers released today.

**Comment by Parovozik** [comment]
Tested on 1.0.639 (https://github.com/benvanik/xenia/commit/257fbfc40846a5301d91c7e512c1be10a19fc649)
Tested on 1.0.685 (https://github.com/benvanik/xenia/commit/cff80bd782b7a0f61a17a2c0020fe839d4f07134)

# Issues:
1.0.639 (Vulkan) // Performance enough good, sound works, save-states works. Partially broken 2D part of graphics. Partially broken 3D part of graphics. On some places color bug. Low brightness (no effect if increase it in settings)
1.0.685 (Vulkan) // 2D part look much better. 3D part - black screen (maybe zero brightness or Something like that) but game works

# Logs:
[1.0.639_Vulkan_Longplay_MEGA.cloud](https://mega.nz/#!w9Zl0TQQ!mtpQYXjTF8T5EIz6dRqBmBvFdrWf0S3gH3eVCc-7ewU)
[1.0.685_Vulkan_Ingame.txt](https://github.com/xenia-project/game-compatibility/files/963118/Oblivion.VK.685.txt)

# Screenshots:
1.0.685_Vulkan
![desktop 04 28 2017 - 01 39 03 02 mp4_snapshot_00 17_ 2017 04 28_02 13 19](https://cloud.githubusercontent.com/assets/19775644/25508133/c829f474-2bb8-11e7-83d4-fbae5a2c8242.jpg)
1.0.685_Vulkan
![desktop 04 28 2017 - 01 39 03 02 mp4_snapshot_02 18_ 2017 04 28_02 13 30](https://cloud.githubusercontent.com/assets/19775644/25508136/cdcde1f6-2bb8-11e7-913c-9bc58ee30aef.jpg)
1.0.685_Vulkan
![desktop 04 28 2017 - 01 39 03 02 mp4_snapshot_02 39_ 2017 04 28_02 13 38](https://cloud.githubusercontent.com/assets/19775644/25508144/d2847e58-2bb8-11e7-87e7-51876d20ce28.jpg)
1.0.639_Vulkan
![desktop 04 28 2017 - 01 42 29 03 mp4_snapshot_00 16_ 2017 04 28_02 14 14](https://cloud.githubusercontent.com/assets/19775644/25508153/dce56128-2bb8-11e7-8141-b8620f724111.jpg)
1.0.639_Vulkan
![desktop 04 28 2017 - 01 42 29 03 mp4_snapshot_02 19_ 2017 04 28_02 14 34](https://cloud.githubusercontent.com/assets/19775644/25508161/e09f4cca-2bb8-11e7-958c-0ad271dc074b.jpg)
1.0.639_Vulkan
![desktop 04 28 2017 - 01 42 29 03 mp4_snapshot_03 58_ 2017 04 28_02 14 59](https://cloud.githubusercontent.com/assets/19775644/25508168/e60c0bee-2bb8-11e7-9b18-a03ae5d67b03.jpg)
1.0.639_Vulkan
![desktop 04 28 2017 - 01 42 29 03 mp4_snapshot_04 42_ 2017 04 28_02 15 08](https://cloud.githubusercontent.com/assets/19775644/25508171/ea830664-2bb8-11e7-90fc-1d29e3937735.jpg)
1.0.639_Vulkan
![desktop 04 28 2017 - 01 42 29 03 mp4_snapshot_08 35_ 2017 04 28_02 16 18](https://cloud.githubusercontent.com/assets/19775644/25508175/ef79aff6-2bb8-11e7-82e0-47b493e5be8f.jpg)
1.0.639_Vulkan
![desktop 04 28 2017 - 01 42 29 03 mp4_snapshot_08 39_ 2017 04 28_02 16 28](https://cloud.githubusercontent.com/assets/19775644/25508180/f4497228-2bb8-11e7-8374-cf789c5f5324.jpg)

# Specs:
CPU Intel i5-3570K @ 3.6 GHz
GPU GTX 1050ti 4GB
RAM 8GB DDR3 (DC)
Windows 10 x64

# Labels:
gpu-corrupt-drawing
state-gameplay

**Comment by bdeimos** [comment]
Tested on (https://github.com/xenia-project/xenia/commit/f89b4626a37dd4d1e5a817909252d3d4dd054299)

Similar state as in last report. Corrupted models and UI elements.

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/2152067/xenia.zip)

![untitled](https://user-images.githubusercontent.com/40533288/42127707-f0cefdae-7ca5-11e8-9fd8-0449c2527b25.jpg)

**Comment by bdelwood** [comment]
Tested on [xenia-project/xenia@eb50eb](https://github.com/xenia-project/xenia/commit/eb50ebd88542547dd2941cec84c46f59353c63a5).

Hardware:
i5-6600K @ 4.4GHz
16GB DDR4
GTX1070

# Log
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/2432577/xenia.zip)


# Issues/Comments
Intro/main menu load fine. Some flickering on logos and background map. Some vector explosions. In-game rendered cut scenes not implemented. A few 3D assets weren't rendered right. 

Otherwise, performance is great, solid 30fps. Few frame drops. Audio is generally good. There is some distortion on position-based vocal audio. 

# Screenshots
![xenia detached_eb50ebd8_sep 29 2018 10_1_2018 12_03_00 am 2 _moment 2](https://user-images.githubusercontent.com/19536361/46269887-1956c400-c512-11e8-9b91-cab6eea6481b.jpg)
![xenia detached_eb50ebd8_sep 29 2018 10_1_2018 12_03_00 am 2 _moment 3](https://user-images.githubusercontent.com/19536361/46269888-1a87f100-c512-11e8-9946-8363ea91805d.jpg)
![xenia detached_eb50ebd8_sep 29 2018 10_1_2018 12_03_00 am 2 _moment 4](https://user-images.githubusercontent.com/19536361/46269889-1bb91e00-c512-11e8-83c6-7856e1fa6adf.jpg)
![xenia detached_eb50ebd8_sep 29 2018 10_1_2018 12_03_00 am 2 _moment 5](https://user-images.githubusercontent.com/19536361/46269890-1cea4b00-c512-11e8-87f4-0d8de4d67d06.jpg)
![xenia detached_eb50ebd8_sep 29 2018 10_1_2018 12_03_00 am 2 _moment](https://user-images.githubusercontent.com/19536361/46269891-1e1b7800-c512-11e8-9c51-7b14bdaaf6df.jpg)


Video of gameplay - https://youtu.be/YX5ywJTdq_k

# Suggested Tags
gpu-corrupt-drawing gpu-shader-errors state-gameplay

**Comment by gandalfthewhite19890404** [comment]
Game works without graphics issues, only when you have torch in hands - fps is drops to 20. But you will get guest crash when you try to get out of sewers to Cirodiil.
